server/application-server/src/main/java/de/tum/in/www1/hephaestus/leaderboard/LeaderboardService.java

```diff
@@ -166,90 +161,8 @@ private int calculateTotalScore(List<PullRequestReview> reviews, int numberOfIss
         double totalScore = reviewsByPullRequestId
             .values()
             .stream()
-            .map(pullRequestReviews -> {
```

Comment:
Can you remove the `[Source]` again from the leaderboard legend?

Reply:
Done in f0412c8.